uvm/impl/com/untangle/uvm/NetworkManagerImpl.java

```diff
@@ -3072,18 +3072,7 @@ public ExecManagerResultReader runTroubleshooting(TroubleshootingCommands comman
      */
     private void convertSettings()
     {
-        try {
-            this.networkSettings.setDhcpRelays( new LinkedList<DhcpRelay>() );
-        } catch (Exception e) {
-            logger.warn("Exception converting Networking Settings",e);
-        }
-
-        // Convert DHCP enabled boolean to enumerated value
-        for( InterfaceSettings interfaceSettings : networkSettings.getInterfaces() ){
-            Boolean dhcpEnabled = interfaceSettings.getDhcpEnabled();
-            interfaceSettings.setDhcpType( dhcpEnabled != null && dhcpEnabled ? InterfaceSettings.DhcpType.SERVER : InterfaceSettings.DhcpType.DISABLED);
-        }
-
+        // For 17.1, peform "free" conversion to set the new dhcpMaxLeases setting to its default value
         this.networkSettings.setVersion( currentVersion );
```

Comment:
Ok... so this causes all of the network settings to be saved ?

Reply:
Yes the convert function causes the network settings to be update and saved.  Unlike the stuff I removed which required "manual" correction (common for these conversions), since this is just adding a new field using the default class value, it's "free" and doesn't require manual conversion.